### PR TITLE
Update Settings Pages

### PR DESCRIPTION
**Pull request by laurapacilio**
Fix style concerns in Terraform Cloud Users, Teams, Organizations section.

### CODE REVIEW

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 1]
```diff
@@ -12,110 +12,97 @@ description: >-
...
-Users must create an account in Terraform Cloud before they can use any of Terraform Cloud's features. Creating an account requires a username, an email address, and a password.
+Creating an account requires a username, an email address, and a password. To use Terraform Cloud, you must create an account through one of the following methods:
 
-If someone invited you to join an existing organization, the invitation email should include a sign-up link.
+- **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
+- **Sign-Up Page:** For Terraform Cloud, go to [https://app.terraform.io/signup/account](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/users.mdx:25]:
```suggestion
- **Sign-Up Page:** For Terraform Cloud, go to [https://app.terraform.io/signup/account](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
```

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 2]
```diff
@@ -12,110 +12,97 @@ description: >-
...
-
-![screenshot: the new organization page](/img/docs/org-new.png)
-
-- If you are the first Terraform Cloud user in your organization, use this page to create your Terraform Cloud organization. See [Organizations][] for more information.
-- If you intend to join an existing organization, do not create a new one; instead, send your email address to one of your organization's owners and ask them to invite you. Once you receive and accept the emailed invitation, you can reload the page to begin using Terraform Cloud.
-
-## Team and Organization Membership
-
-To join an organization, you must be invited by an owner of that organization. In order to have any permissions with that organization, an owner will need to add you to one or more teams. For more information, see [Organization Settings: Users][invite] and [Managing Team Membership](/cloud-docs/users-teams-organizations/teams#managing-team-membership).
+An organization owner must [invite you to join an organization](/cloud-docs/users-teams-organizations/organizations#users). After you join, the organization owner must also [add you to one or more teams](/cloud-docs/users-teams-organizations/teams#managing-team-membership).
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/users.mdx:29]:
```suggestion
An organization owner must [invite you to join their organization](/cloud-docs/users-teams-organizations/organizations#users). After you join, the organization owner must also [add you to one or more teams](/cloud-docs/users-teams-organizations/teams#managing-team-membership).
```

I know this is obvious, but I feel that this change gets rid of a small understanding loophole where users might believe that an invite from any org owner could get them access to other orgs which aren't associated with the owner who invited them.

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 3]
```diff
@@ -12,110 +12,97 @@ description: >-
...
+API tokens are required for the following tasks:
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+- Authenticating with the [Terraform Cloud API](/cloud-docs/api-docs). API calls require an `Authorization: Bearer <TOKEN>` HTTP header.
+- Authenticating with the [Terraform Cloud CLI integration](/cli/cloud/settings) or the [`remote` backend](/language/settings/backends/remote). These require a token in the CLI config file or in the backend configuration.
+- Using [private modules](/cloud-docs/registry/using) in command-line runs on local machines. This requires [a token in the CLI config file](/cloud-docs/registry/using#authentication).
 
-Since users can be members of multiple organizations, user tokens work with any organization their user belongs to.
+Protect your tokens carefully because they can do anything your user account can. For example, if you belong to a team with permission to read and write variables for a workspace, your API token can edit variables in that workspace. Refer to [permissions](/cloud-docs/users-teams-organizations/permissions) for more details.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/users.mdx:93]:
```suggestion
Protect your tokens carefully because they contain the same permissions as your user account. For example, if you belong to a team with permission to read and write variables for a workspace, another user could use your API token to authenticate as your user account and also edit variables in that workspace. Refer to [permissions](/cloud-docs/users-teams-organizations/permissions) for more details.
```
I just want to rephrase this as another option which clicked a bit easier in my head, referring to the API token as a tool with permissions instead of it being the actor making the changes.

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 4]
```diff
@@ -13,179 +13,188 @@ description: >-
...
+Terraform Cloud displays your current organization in the left of the top navigation bar. To select an organization:
 
-![screenshot: the organization switcher menu](/img/docs/org-nav.png)
+1. Click the current organization name to view a list of all the organizations where you are a member.
+1. Click an organization to select it. Terraform Cloud displays list of workspaces within that organization.
 
 ## Joining and Leaving Organizations
 
-To join an organization, you must be invited by one of its [owners][] and must accept the emailed invitation. See [Organization Settings: Users](#users) below for details.
+To join an organization, you must be invited by one of its [owners][] and must accept the emailed invitation. Refer to [Organization Settings: Users](#users) for details.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:33]:
```suggestion
To join an organization, you must be invited by one of its owners and must accept the emailed invitation. Refer to [Organization Settings: Users](#users) for details.
```
Not sure if those brackets were meant to be included or not.

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 5]
```diff
@@ -13,179 +13,188 @@ description: >-
...
 
 ## Creating Organizations
 
-Users can create new organizations by clicking the "Create new organization" button in the organization switcher menu. The new organization page also displays automatically when the currently logged-in user does not belong to any organizations, as when first logging in as a new user.
-
-![screenshot: the new organization page](/img/docs/org-new.png)
+On Terraform Enterprise, administrators can restrict your ability to create organizations. Refer to [Administration: General Settings](/enterprise/admin/application/general#organization-creation) for details.
 
-To create a new organization, provide a unique name and a primary contact email address. Organization names can include numbers, letters, underscores (`_`), and hyphens (`-`).
+On Terraform Cloud, any user can create a new organization. If you do not belong to any organizations, Terraform Cloud prompts you to create one the first time you log in. Otherwise, to create an organization:
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:43]:
```suggestion
On Terraform Cloud, any user can create a new organization. If you do not belong to any organizations, Terraform Cloud prompts you to create one the first time you log in. To create an organization:
```
It might just be me, but I can't see the two clauses that 'otherwise' is linking here.

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 6]
```diff
@@ -13,179 +13,188 @@ description: >-
...
 
-The contents of the organization settings depends on the current user's permissions within the organization:
+The contents of the organization settings depends on your permissions within the organization. All users can review the organization's contact email, view the membership of any teams they belong to, and view the organization's authentication policy. [Organization owners][owners] can view and manage the entire list of organization settings. Refer to [Organization Permissions](/cloud-docs/users-teams-organizations/permissions#organization-permissions) for details.
 
-- Users with permission to manage VCS settings or manage policies can manage the settings they have access to. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
-- [Organization owners][owners] can view and manage the entire list of organization settings. Most settings cannot be delegated, and are only available to owners.
-- Other users can use this section to view the organization's contact email, view the membership of any teams they belong to, and view the organization's authentication policy.
+You may be able to manage the following types of organization permissions.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:59]:
```suggestion
You may be able to manage the following organization permissions.
```

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 7]
```diff
@@ -13,179 +13,188 @@ description: >-
...
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-Most of the organization settings are documented near the specific workflows they enable. What follows is a brief summary with links to more relevant sections of the documentation.
+### Organization Settings
 
-### General
+#### General
 
-The general settings page is shown to all users in an organization, and displays the organization's name and contact email.
+Review the organization name and contact email. Organization owners can also  change the organization name and contact email or delete the organization.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:67]:
```suggestion
Review the organization name and contact email. Organization owners can also change the organization name and contact email or delete the organization.
```

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 8]
```diff
@@ -13,179 +13,188 @@ description: >-
...
 
-Organization owners can use this page to change the organization's name and contact email or delete the organization.
+!> **Warning:** Deleting or renaming an organization can be very disruptive. We strongly recommend against deleting or renaming organizations with active members.
 
-#### Renaming an Organization
+To rename an organization that manages infrastructure:
 
-!> **Warning:** Renaming an organization can be very disruptive. With the exception of new organizations with no collaborators, we strongly recommend against renaming organizations.
+1.  Alert all members of the organization about its impending deletion.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:73]:
```suggestion
1. Alert all members of the organization about its impending deletion.
```

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 9]
```diff
@@ -13,179 +13,188 @@ description: >-
...
-1. Unlock workspaces and resume normal operations.
+#### Tags
+
+Review a list of tags for all resources across the organization. When you delete a tag from this page, Terraform Cloud removes it from all resources.
 
-### Plan & Billing
+#### Teams
 
-The plan and billing page allows organization owners to upgrade to one of Terraform Cloud's paid plans, downgrade to a free plan, or begin a free trial of paid features. It also displays any invoices for previous plan payments.
+-> **Note:** Team management is a paid feature, available as part of the **Team** upgrade plan. Free organizations only include an owners team that can include up to five members. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:90]:
```suggestion
-> **Note:** Team management is a paid feature, available as part of the **Team** upgrade plan. Free plan organizations only include an owners team that can include up to five members. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
```

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 10]
```diff
@@ -13,179 +13,188 @@ description: >-
...
 
-### Tags
+All users in an organization can access the **Teams** page, which displays a list of [teams][] within the organiation. This excludes [secret teams](/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:92]:
```suggestion
All users in an organization can access the **Teams** page, which displays a list of teams within the organiation. This excludes [secret teams](/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.
```

I wasn't sure if those brackets should be included or not.

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 11]
```diff
@@ -13,179 +13,188 @@ description: >-
...
 
-This page shows the list of tags used for all resources across the organization. Tags can be added directly to workspaces. Tags deleted here will be removed from all other resources, and can be deleted in bulk.
+Organization owners can also create and delete teams, manage team membership, and manage team API tokens. Remember that users must accept an invitation to the organization before you can add them to teams.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:94]: I really like that we're reminding them here, because they do often forget!

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 12]
```diff
@@ -13,179 +13,188 @@ description: >-
...
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-![Screenshot: the tags list, showing a list of currently used tags.](/img/docs/tag-management.png)
+#### Users
 
-### Teams
+Organization owners can invite Terraform Cloud users into the organization, cancel invitations, and remove existing members.
 
--> **Note:** Team management is a paid feature, available as part of the **Team** upgrade package. Free organizations only include an owners team, which can include up to five members. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+The list of users has one tab for active users and one tab for invited users who have not yet accepted their invitations. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships. Use the **Search by username or email** field to filter these lists.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:102]:
```suggestion
The list of users is separated into one tab for active users and one tab for invited users who have not yet accepted their invitations. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships. Use the **Search by username or email** field to filter these lists.
```
Is the list of invited-but-not-accepted users worth mentioning in terms of contents?

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 13]
```diff
@@ -13,179 +13,188 @@ description: >-
...
+-> **Note:** Cost estimation is a paid feature, available as part of the **Team & Governance** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
 
-User invitations are always sent by email; you cannot invite someone using their Terraform Cloud username.
+Enable and disable the [cost estimation](/cloud-docs/cost-estimation) feature for all workspaces.
 
--> **Note:** All permissions in Terraform Cloud are managed via teams. Although users can join an organization without belonging to any teams, they won't be able to do anything until they are also added to a team. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
+#### Policies
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+This is a deprecated interface for managing Sentinel policies. Use the policy sets page instead.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:131]:
```suggestion

```
Is this even worth keeping if it is deprecated?

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 14]
```diff
@@ -13,179 +13,188 @@ description: >-
...
+### Security
 
-The authentication page allows owners to determine when users must reauthenticate. It also allows owners to require [two-factor authentication](/cloud-docs/users-teams-organizations/2fa) for all members of the organization.
+#### Agents
 
-### SSH Keys
+-> Note: Terraform Cloud Agents are a paid feature, available as part of the **Business** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
 
-The SSH keys page manages any keys necessary for cloning Git-based module sources during Terraform runs. It does not manage keys used for accessing a connected VCS provider. See [SSH Keys for Cloning Modules](/cloud-docs/workspaces/settings/ssh-keys) for more information.
+Create and manage [Terraform Cloud agent pools](/cloud-docs/agents). Terraform Cloud Agents let Terraform Cloud to communicate with isolated, private, or on-premises infrastructure.  This is useful for on-premises infrastructure types such as vSphere, Nutanix, OpenStack, enterprise networking providers, and infrastructure within a protected enclave.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:151]:
```suggestion
Create and manage [Terraform Cloud agent pools](/cloud-docs/agents). Terraform Cloud Agents let Terraform Cloud communicate with isolated, private, or on-premises infrastructure. This is useful for on-premises infrastructure types such as vSphere, Nutanix, OpenStack, enterprise networking providers, and infrastructure within a protected enclave.
```

File: content/cloud-docs/users-teams-organizations/organizations.mdx [hunk 15]
```diff
@@ -13,179 +13,188 @@ description: >-
...
+Workspaces that use part of a shared repository do not typically run plans for changes that do not affect their files. This includes [speculative plans](/cloud-docs/run/remote-operations#speculative-plans) on pull requests. Since **pending** status checks can block pull requests, workspaces automatically send passing commit statuses for any PRs that do not affect their files.
 
-Refer to [Run Tasks](/cloud-docs/workspaces/settings/run-tasks) for more information.
+You can disable this behavior if it creates too many status checks to your VCS provider. You may want to do this if you have a large number of workspaces sharing one VCS repository.
 
-### VCS Providers
+#### VCS Events
 
-The VCS providers page is used for configuring VCS providers for use within the organization. It is available to users with permission to manage VCS settings. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
+-> **Note:** This feature is in beta.
```
Comment (duplo83) [inline content/cloud-docs/users-teams-organizations/organizations.mdx:181]:
```suggestion
-> **Note:** This feature is in beta as of May 2022.
```
On features like this, is it worth mentioning a timestamp of some kind to bookmark when the docs were made, or is the feature's release status being tracked in some other channel?

**Review by duplo83 — APPROVED** [review]
I think the changes outline here are all definite improvements. They add clarity and important emphasis in areas that have been pain points for customers. 
The edits I suggested are minimal and very specific, mostly for clarity. 

Great work, Laura!

File: content/cloud-docs/users-teams-organizations/organizations.mdx (hunk 13 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/organizations.mdx:131]: That's a fair point, Dane. It's still in the UI, so I think we do need to document it so folks aren't confused. When they take it out of the UI, I can take it out of the docs!

File: content/cloud-docs/users-teams-organizations/organizations.mdx (hunk 10 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/organizations.mdx:92]: ahh that's a fair question! This is a weird linking pattern that is kind of legacy throughout the docs. So in this case yes, it should be. They define what this link is at the top of the page and then we can reuse it throughout without having to re-enter the link path each time (it's pretty convenient actually).

File: content/cloud-docs/users-teams-organizations/organizations.mdx (hunk 8 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/organizations.mdx:73]: yeah it is!! thank you

File: content/cloud-docs/users-teams-organizations/organizations.mdx (hunk 4 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/organizations.mdx:33]: same as below! they are supposed to be here, but this is an unusual pattern, so I appreciate you calling it out!

File: content/cloud-docs/users-teams-organizations/users.mdx (hunk 3 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/users.mdx:93]: this is much much better. Thank you! I phoned it in and didn't rewrite as much as I should have here. I appreciate you!

File: content/cloud-docs/users-teams-organizations/organizations.mdx (hunk 5 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/organizations.mdx:43]: yeah this is fair. I think I was just trying to be like "you will do this automatically if you don't already have an org" but I feel like it was poorly done. Good edit!

File: content/cloud-docs/users-teams-organizations/users.mdx (hunk 2 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/users.mdx:29]: yes! thank you!

File: content/cloud-docs/users-teams-organizations/organizations.mdx (hunk 15 above)
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/organizations.mdx:181]: this is an interesting question. We don't typically put times into the docs like this if we can help it. Theoretically, the docs should be up-to-date. BUT thank you for flagging this because I actually do want to flag whether this is still in beta.....there are some inconsistencies in the UI

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 16]
```diff
@@ -12,110 +12,97 @@ description: >-
 
 # Users
 
-User accounts belong to individual people. A user can be a member of multiple [organizations][].
+User accounts belong to individual people. Each user can be part of one or more [teams](/cloud-docs/users-teams-organizations/teams), which are granted permissions on workspaces within an organization. A user can be a member of multiple [organizations][].
```
Comment (judithpatudith) [inline content/cloud-docs/users-teams-organizations/users.mdx:15]:
```suggestion
User accounts belong to individual people. Each user can be part of one or more [teams](/cloud-docs/users-teams-organizations/teams), which are granted permissions on workspaces within an organization. A user can be a member of multiple [organizations][].
```

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 17]
```diff
@@ -12,110 +12,97 @@ description: >-
...
+Creating an account requires a username, an email address, and a password. To use Terraform Cloud, you must create an account through one of the following methods:
 
-If someone invited you to join an existing organization, the invitation email should include a sign-up link.
+- **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
+- **Sign-Up Page:** For Terraform Cloud, go to [https://app.terraform.io/signup/account](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
```
Comment (judithpatudith) [inline content/cloud-docs/users-teams-organizations/users.mdx:25]:
```suggestion
- **Sign-Up Page:** For Terraform Cloud, go to [`https://app.terraform.io/signup/account`](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
```

File: content/cloud-docs/users-teams-organizations/teams.mdx [hunk 18]
```diff
@@ -13,123 +13,138 @@ description: >-
...
+1. Click **Delete [team name]** at the bottom of the page. The **Deleting team "[team name]"** box appears.
+1. Click **Yes, delete team** to permanently delete the team and all of its data from Terraform Cloud.
 
-To delete a team, go to the target team's settings page and click the "Delete TEAM NAME" button.
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
 
 ### Managing Team Membership
 
-Organization owners can use a team's settings page to add and remove users from the team.
+Team structure often resembles your company's group structure. Refer to the [Terraform Recommended Practices guide](/cloud-docs/guides/recommended-practices) for a discussion of how team structure interacts with the structure of your Terraform configurations and the corresponding infrastructure.
```
Comment (robin-norwood) [inline content/cloud-docs/users-teams-organizations/teams.mdx:87]:
```suggestion
Team structure often resembles your company's organizational structure. Refer to the [Terraform Recommended Practices guide](/cloud-docs/guides/recommended-practices) for a discussion of how team structure interacts with the structure of your Terraform configurations and the corresponding infrastructure.
```

Maybe more clear?

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 19]
```diff
@@ -12,110 +12,97 @@ description: >-
...
 
-Protect your tokens carefully, because they can do anything your user account can. For example, if you belong to a team with permission to read and write variables for a workspace, your API token can edit variables in that workspace. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
+API tokens are required for the following tasks:
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+- Authenticating with the [Terraform Cloud API](/cloud-docs/api-docs). API calls require an `Authorization: Bearer <TOKEN>` HTTP header.
+- Authenticating with the [Terraform Cloud CLI integration](/cli/cloud/settings) or the [`remote` backend](/language/settings/backends/remote). These require a token in the CLI config file or in the backend configuration.
+- Using [private modules](/cloud-docs/registry/using) in command-line runs on local machines. This requires [a token in the CLI config file](/cloud-docs/registry/using#authentication).
```
Comment (robin-norwood) [inline content/cloud-docs/users-teams-organizations/users.mdx:91]:
```suggestion
- Using [private modules](/cloud-docs/registry/using) in command-line runs on local machines. This requires [a token in the CLI configuration file](/cloud-docs/registry/using#authentication).
```

**Review by robin-norwood — APPROVED** [review]
lgtm! A couple of minor things.

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 20]
```diff
@@ -12,110 +12,97 @@ description: >-
...
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-User invitations are sent by email. If the invited email address matches an existing Terraform Cloud account, the invitee can join the organization with that account; if not, they must create a new account for working with that organization.
+Terraform Cloud sends user invitations by email. If the invited email address matches an existing Terraform Cloud account, the invitee can join the organization with that account. Otherwise, they must create a new account to work with that organization.
 
 ## Site Admin Permissions
 
-On Terraform Enterprise instances, some user accounts have a special "site admin" permission that allows administration of the entire instance.
+On Terraform Enterprise instances, some user accounts have a special site admin permission that allows them to administrate the entire instance.
```
Comment (judithpatudith) [inline content/cloud-docs/users-teams-organizations/users.mdx:37]:
```suggestion
On Terraform Enterprise instances, some user accounts have a special site admin permission that allows them to administer the entire instance.
```

File: content/cloud-docs/users-teams-organizations/users.mdx [hunk 21]
```diff
@@ -12,110 +12,97 @@ description: >-
...
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/users.mdx]: thank you!! Fixed - judith also flagged it :)
